.gitignore

```diff
@@ -16,3 +16,4 @@ Thumbs.db
 *.suo
 .vs/
 *.xproj.user
+.idea
```

Comment:
This is Jetbrains Project Rider. It's a standalone C# editor with ReSharper. I develop on Linux so no visual studio for me